src/MoBi.UI/Views/EditParameterListView.cs

```diff
@@ -215,10 +216,30 @@ public void SetVisibility(PathElement pathElement, bool isVisible)
          _pathBinder.SetVisibility(pathElement, isVisible);
       }
 
+      public IReadOnlyList<ParameterDTO> SelectedParameters
+      {
+         get { return _gridView.GetSelectedRows().Select(rowHandle => _gridViewBinder.ElementAt(rowHandle)).ToList(); }
+         set
```

Comment:
THis code should be moved in one single place. This is exactly the same in PKSim and MoBi. I thin GridViewBinder would be  a good place but I need to make sure that this works before moving it. Can be done later